roles/default/kiali-deploy/defaults/main.yml

```
@@ -163,6 +167,9 @@ kiali_defaults:
       cache_duration: 7
       cache_enabled: true
       cache_expiration: 300
+      component_status:
+        is_core: true
+        health_check_url: ""
```

all peer settings should be sorted alphabetically. "h" before "i" :)

```suggestion
        health_check_url: ""
        is_core: true
```